### DISCUSSION TITLE
Search Bar

### DISCUSSION THREAD

**Discussion by Bryan-Fagan** [dropways/deskapp#34] [Q&A]
Hi,

I want to start off with this template is great.  For my site, I'm using a lot of the things you showed with an MKDocks theme.  How do I like the search bar to the built in MKDocs search plugin?

**Comment by dropways** [comment]
Hello @GetFinanceIndependent 

I did not have any experience with MkDocs plugin.

**Reply by Bryan-Fagan** [reply]
I guess the other question is what will make the search bar actual work?  Is there something beyond the template?  I wasn't able to get it in the working template site either. I'm not sure if that was by design or something else.
